scripts/mirror_releases.sh

```diff
@@ -2,7 +2,7 @@
 set -o errexit -o nounset
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
-version="$(curl --silent "https://registry.npmjs.org/webpack/latest" | jq --raw-output ".version")"
+version="${1:-$(curl --silent "https://registry.npmjs.org/webpack/latest" | jq --raw-output ".version")}"
```

Comment:
What was the reason for this?

Reply:
It makes it consistent with the other repos which allow you to call `./script/mirror_releases.sh 1.2.3`(with a version param)